packages/lib/gpc/src/gpcCompile.ts

```diff
@@ -760,6 +768,17 @@ export function compileVerifyConfig(
   // since all the signals involved are public.
   const circuitGlobalInputs = compileProofGlobal(verifyRevealed);
 
+  // Virtual entry inputs
+  // TODO(POD-P3): Replace this with actual inputs.
```

Comment:
Interesting that you marked this as P3, but marked the TODO to delete the functions called here as P2.  Seems impossible to actually do them in that order.  Was there an intent here, or just copy/paste?
I think everything related to this feature can be P3, to the extent these priorities are still relevant.

Reply:
P2 sounds right.